dependencies/pip/requirements.in

```diff
@@ -13,7 +13,7 @@
 -e git+https://github.com/kobotoolbox/ssrf-protect@9b97d3f0fd8f737a38dd7a6b64efeffc03ab3cdd#egg=ssrf_protect
 
 # Regular PyPI packages
-Django>=2.2,<2.3
+Django==3.2
```

Comment:
Should this be `>=3.2,<3.3`? This version exactly would have known security vulnerabilities in it.

Reply:
I thought that `==3.2` would always have installed the latest patch version. I'm gonna change it to `>=3.2, <3.3` anyway.